### PR TITLE
jsx template, removed a11y functions which were not needed

### PR DESCRIPTION
**Pull request by ethan-lp**

### CODE REVIEW

File: js/flipcardView.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
+import Adapt from 'core/js/adapt';
+import ComponentView from 'coreViews/componentView';
+import _ from 'underscore';
+
+class Flipcard extends ComponentView {
```
Comment (cahirodoherty-learningpool) [inline js/flipcardView.js]:
```suggestion
class FlipcardView extends ComponentView {
```

File: js/flipcardView.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  }
+
+  // This function will check or set the completion status of current component.
+  checkCompletionStatus() {
+    if (this.getVisitedItems().length !== this.model.get('_items').length) return;
+    this.setCompletionStatus();
+  }
+}
+
+Flipcard.template = 'flipcard.jsx';
```
Comment (cahirodoherty-learningpool) [inline js/flipcardView.js]:
```suggestion
FlipcardView.template = 'flipcard.jsx';
```

File: js/flipcardView.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+export default Flipcard;
```
Comment (cahirodoherty-learningpool) [inline js/flipcardView.js]:
```suggestion
export default FlipcardView;
```

File: templates/flipcard.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            ])}
+            key={index}
+          >
+            <div
+              className='flipcard__item-face flipcard__item-front'
+              role='button'
+            >
+              <img
+                className='flipcard__item-frontImage'
+                src={frontImage.src}
```
Comment (cahirodoherty-learningpool) [inline templates/flipcard.jsx]:
```suggestion
                src={frontImage?.src}
```

File: templates/flipcard.jsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                aria-label={frontImage.alt}>
```
Comment (cahirodoherty-learningpool) [inline templates/flipcard.jsx]:
```suggestion
                aria-label={frontImage?.alt}>
```

File: js/flipcardView.js [hunk 6]
```diff
@@ -0,0 +1,191 @@
+import Adapt from 'core/js/adapt';
+import ComponentView from 'coreViews/componentView';
+import _ from 'underscore';
```
Comment (eleanor-heath) [inline js/flipcardView.js]: this import isnt needed

File: js/flipcardView.js [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+      'click .flipcard__item': 'onClickFlipItem',
+      'keypress .flipcard__item': 'onClickFlipItem'
+    };
+  }
+
+  preRender() {
+    this.model.set('_items',
+      this.model.get('_items').map(obj => ({
+        ...obj
+      }))
```
Comment (eleanor-heath) [inline js/flipcardView.js:17]: minor: missing semicolon

Comment (ethan-lp) [reply js/flipcardView.js:17]: The semicolon is on the line below, putting one here would cause a syntax error

**Comment by eleanor-heath** [comment]
Hey, can you double check everything that was tested on ADAPT-8325 - https://github.com/LearningPool/adapt-contrib-flipcard/pull/47 and ADAPT-7585 - https://github.com/LearningPool/adapt-contrib-flipcard/pull/45 still works with this because I did a lot of work around accessibility for flipcard and a11y was needed at the time to ensure accessibility was working across different screen readers and to ensure the screen reader will read the content behind flipcard without requiring the learner to flip the card first (by pressing the space bar). Also need to make sure single flip and all flip still functions as expected and you should be able to tab to the flipcard items.

File: js/flipcardView.js (hunk 7 above)
Comment (eleanor-heath) [reply js/flipcardView.js:17]: my mistake!

**Review by eleanor-heath — APPROVED** [review]

File: templates/flipcard.jsx [hunk 8]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { templates, classes, compile } from 'core/js/reactHelpers';
+
+export default function flipcard(props) {
+  const {} = props;
```
Comment (cahirodoherty-learningpool) [inline templates/flipcard.jsx]: There's nothing getting deconstructed here, so can delete the line

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by eleanor-heath — APPROVED** [review]

File: templates/flipcard.hbs [hunk 9]
```diff
@@ -1,28 +0,0 @@
...
-                  {{{backTitle}}}
-                </div>
-              {{/if}}
-              {{#if backBody}}
-                <div class="flipcard__item-back-body">
-                  {{{compile backBody}}}
-                </div>
-              {{/if}}
-            </div>
-          </div>
```
Comment (eleanor-heath) [inline templates/flipcard.hbs:24]: #

**Pull request merged**